### PR TITLE
DOC-7722: N1QL transactions beta fixes

### PR DESCRIPTION
**Pull request by simon-dew**
The following draft documentation is ready for review:

* [BEGIN TRANSACTION](https://docs-staging.couchbase.com/server/7.0/n1ql/n1ql-language-reference/begin-transaction.html)
* [SET TRANSACTION](https://docs-staging.couchbase.com/server/7.0/n1ql/n1ql-language-reference/set-transaction.html)
* [SAVEPOINT](https://docs-staging.couchbase.com/server/7.0/n1ql/n1ql-language-reference/savepoint.html)
* [ROLLBACK TRANSACTION](https://docs-staging.couchbase.com/server/7.0/n1ql/n1ql-language-reference/rollback-transaction.html)
* [COMMIT TRANSACTION](https://docs-staging.couchbase.com/server/7.0/n1ql/n1ql-language-reference/commit-transaction.html)

Docs issue: [DOC-7722](https://issues.couchbase.com/browse/DOC-7722)

### CODE REVIEW

File: modules/n1ql/pages/n1ql-language-reference/begin-transaction.adoc [hunk 1]
```diff
@@ -66,27 +69,23 @@ The statement returns an object containing the following property.
 
 txid::
 The transaction ID.
-You must supply this transaction ID with any subsequent statements that form part of the same transaction.
+
+If you are using the Query Workbench or the Query REST API, you must set the {txid}[txid] query parameter to specify the transaction ID for any subsequent statements that form part of the same transaction.
+
+If you are using the cbq shell, you don't need to specify the transaction ID for any subsequent statements that form a part of the same transaction.
+Once you have started a transaction, all eligible statements within the cbq shell session are assumed to be part of the same transaction until you rollback or commit the transaction.
```
Comment (sitaramv) [inline modules/n1ql/pages/n1ql-language-reference/begin-transaction.adoc]: remove eligible

File: modules/n1ql/pages/n1ql-language-reference/commit-transaction.adoc [hunk 2]
```diff
@@ -22,10 +22,15 @@ The `COMMIT TRANSACTION` statement enables you to commit a transaction.
 The `COMMIT TRANSACTION` statement enables you to commit an ACID transaction.
 Refer to {transactions}[Transactions] for further information.
 
-You must set the {txid}[txid] query parameter to specify the transaction that you want to commit.
+If you are using the Query Workbench or the Query REST API, you must set the {txid}[txid] query parameter to specify the transaction ID.
+
+If you are using the cbq shell, you don't need to specify the transaction ID.
+Once you have started a transaction, all eligible statements within the cbq shell session are assumed to be part of the same transaction until you rollback or commit the transaction.
```
Comment (sitaramv) [inline modules/n1ql/pages/n1ql-language-reference/commit-transaction.adoc]: remove eligible

File: modules/n1ql/pages/n1ql-language-reference/rollback-transaction.adoc [hunk 3]
```diff
@@ -24,10 +24,16 @@ You can rollback the entire transaction, or rollback to a previous savepoint.
 Refer to {transactions}[Transactions] for further information.
 
 This statement may only be used within a transaction.
-You must set the {txid}[txid] query parameter to specify the transaction.
+
+If you are using the Query Workbench or the Query REST API, you must set the {txid}[txid] query parameter to specify the transaction ID.
+
+If you are using the cbq shell, you don't need to specify the transaction ID.
+Once you have started a transaction, all eligible statements within the cbq shell session are assumed to be part of the same transaction until you rollback or commit the transaction.
```
Comment (sitaramv) [inline modules/n1ql/pages/n1ql-language-reference/rollback-transaction.adoc]: remove eligible

File: modules/n1ql/pages/n1ql-language-reference/begin-transaction.adoc (hunk 1 above)
Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/begin-transaction.adoc]: Done

File: modules/n1ql/pages/n1ql-language-reference/commit-transaction.adoc (hunk 2 above)
Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/commit-transaction.adoc]: Done

File: modules/n1ql/pages/n1ql-language-reference/rollback-transaction.adoc (hunk 3 above)
Comment (simon-dew) [reply modules/n1ql/pages/n1ql-language-reference/rollback-transaction.adoc]: Done

**Review by kaminijagtiani — APPROVED** [review]

**Review by kaminijagtiani — APPROVED** [review]
Looks good now.Thanks

**Comment by kaminijagtiani** [comment]
Can we also add a complete example of a transaction in cbq from BEGIN to COMMIT and BEGIN to ROLLBACK?
I had pasted it in slack yesterday

**Comment by simon-dew** [comment]
> Can we also add a complete example of a transaction in cbq from BEGIN to COMMIT and BEGIN to ROLLBACK?

My feeling is, that example would be too big to go into any of the reference pages for individual commands. But I can see that a complete example would be useful. I think it would go better in a tutorial or guided example on transactions in N1QL as a whole. I'll create a separate issue.

**Comment by simon-dew** [comment]
Update links to staging docs

**Pull request merged**